201-sql-auditing-server-policy-to-blob-storage/azuredeploy.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "sqlServerName": {
+        "type": "string",
+        "defaultValue": "[concat('server-', uniqueString(resourceGroup().id, deployment().name))]",
+        "metadata": {
+          "description": "Name of the SQL server"
+        }
+      },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "sqlAdministratorLogin": {
+      "type": "string",
+      "metadata": {
+        "description": "The administrator username of the SQL Server."
+      }
+    },
+    "sqlAdministratorLoginPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The administrator password of the SQL Server."
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "defaultValue": "[concat('sqlaudit',uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "The name of the auditing storage account."
+      }
+    }
+  },
+  "variables": {
+    
+  },
+  "resources": [{
+    "type": "Microsoft.Storage/storageAccounts",
+    "name": "[parameters('storageAccountName')]",
+    "apiVersion": "2017-10-01",
+    "location": "[parameters('location')]",
+    "sku": {
+      "name": "Standard_LRS"
+    },
+    "kind": "Storage",
+    "properties": {
+    }
+  },
+  {
+    "type": "Microsoft.Sql/servers",
+    "apiVersion": "2015-05-01-preview",
+    "location": "[parameters('location')]",
+    "name": "[parameters('sqlServerName')]",
+    "properties": {
+      "administratorLogin": "[parameters('sqlAdministratorLogin')]",
+      "administratorLoginPassword": "[parameters('sqlAdministratorLoginPassword')]",
+      "version": "12.0"
+    },
+    "tags": {
+      "DisplayName": "[parameters('sqlServerName')]"
+    },
+    "resources": [{
+      "apiVersion": "2017-03-01-preview",
+      "type": "auditingSettings",
+      "name": "DefaultAuditingSettings",
+      "dependsOn": ["[parameters('sqlServerName')]",
+      "[parameters('storageAccountName')]"],
+      "properties": {
+        "state": "Enabled",
+        "storageEndpoint": "[concat('https://',parameters('storageAccountName'),'.blob.core.windows.net')]",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#referencing-resource-properties) for how to handle storage account uris